src/lib/maps/Client.ts

```diff
@@ -263,3 +279,53 @@ export interface MapInstance {
     };
   };
 }
+
+export interface MapDataviewsOptions {
+  /**
+   * column name to aggregate by
+   */
+  column: string;
+
+  /**
+   * operation to perform
+   */
+  aggregation: AggregationType;
+
+  /**
+   * operation to perform
+   */
+  operation: AggregationType;
+
+  /**
+   * The num of categories
+   */
+  categories?: number;
+
+  /**
+   * Column value to aggregate.
+   * This param is required when
+   * `aggregation` is different than "count"
+   */
+  operationColumn?: string;
+
+  /**
+   * [Maps API parameter name]
+   * Same as operationColumn but this is the
+   * name which is used by Maps API as parameter
+   */
+  aggregationColumn?: string;
+
+  /**
+   * Bounding box to filter data
+   */
+  bbox?: number[];
+}
+
+export enum AggregationType {
```

Comment:
I think we have them already declared in another modules, and we should avoid having enumerations with the same values.

Reply:
nice catch!